ArcGISRuntimeSDKQt_CppSamples/Layers/ManageOperationalLayers/ManageOperationalLayers.qml

```diff
@@ -76,7 +76,7 @@ Item {
             model: sample.layerListModel
             delegate: Component {
                 RowLayout {
-                    width: parent.width
+                    width: 200
```

Comment:
instead of hardcoding it, does it work to specify by id instead of parent - `layersList.width`?